gobblin-runtime/src/test/java/gobblin/runtime/mapreduce/MRJobLauncherTest.java

```diff
@@ -228,12 +228,11 @@ public void testLaunchJobWithMultipleDatasets() throws Exception {
    * Seems setting mapreduce.map.failures.maxpercent=100 does not prevent Hadoop2's LocalJobRunner from
    * failing and aborting a job if any mapper task fails. Aborting the job causes its working directory
    * to be deleted in {@link GobblinOutputCommitter}, which further fails this test since all the output
-   * {@link gobblin.runtime.TaskState}s are deleted. It works fine in Hadoop1 though by setting
-   * mapred.max.map.failures.percent=100. There may be a bug in Hadoop2's LocalJobRunner.
+   * {@link gobblin.runtime.TaskState}s are deleted. There may be a bug in Hadoop2's LocalJobRunner.
    *
    * Also applicable to the two tests below.
    */
-  @Test(groups = { "Hadoop1Only" })
+  @Test(groups = { "ignore" })
```

Comment:
I'm not sure. According to the comments for the tests, they were disabled for Hadoop 2.x.x because of an apparent bug in Hadoop 2.x.x's LocalJobRunner. Hopefully, one day that bug will be fixed, in which case we can re-enabled these tests.